docs/articles/clustering/cluster-singleton.md

```diff
@@ -35,6 +43,65 @@ This pattern may seem to be very tempting to use at first, but it has several dr
 
 Especially the last point is something you should be aware of — in general when using the Cluster Singleton pattern you should take care of downing nodes yourself and not rely on the timing based auto-down feature.
 
+## An Example (Simplified API)
+
+Any `Actor` can be run as a singleton. E.g. a basic counter:
+
+```csharp
```

Comment:
Generally, we prefer to use DocFx's `[code]` blocks that reference a runnable source file instead of traditional markdown blocks - see https://getakka.net/community/contributing/documentation-guidelines.html#code-samples-must-use-code-references for examples on how to do this.

Reason why we care: if the sample is updated, the documentation is also automatically updated too.

Reply:
I'm going to merge this in now and we can update those docs to reference your unit tests' code in a separate PR.